README.md

```diff
@@ -20,6 +20,8 @@ Summary of supported devices and features
 |Roller shutter mode|✗|✗|✗|✗|✗|✗|✓|✗|✗|
 |Power measurement|✗|✓|-|✓|✓|✗|✓|✗|-|
 |RGB(W)|✗|✗|✗|✗|✗|✗|✗|✗|✓|
+|CCT (WW,CW)|✗|✗|✗|✗|✗|✗|✗|✗|✓|
```

Comment:
since this hasn't been released yet, let's not update it now. create a separate PR or ask Timo to add it to #892